fdmi/plugins/fdmi_plugin_st.sh

```diff
@@ -58,7 +58,7 @@ do_some_kv_operations()
 			    -f $M0T1FS_PROC_ID -s "
 
 		echo "Let's create an index and put {somekey:somevalue}"
-		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
```

Comment:
${MOTR_PARAM} can not be double quoted.

It is treated as multiple arguments intentionally.

If double quoted, it will be treated as a single argument. This is wrong.